### DISCUSSION TITLE
Randomizers

### DISCUSSION THREAD

**Discussion by pehsa** [opengaming/osgameclones#2549] [General]
I saw the Wind Waker Randomizer PR and I know so many games has a randomizer whose are OS softwares. Thanks to the speedrunning community here one (maybe outdated) list:

https://docs.google.com/spreadsheets/d/1rXgXxS1W9AaEhhpxfGjNyLuxC_P7Xa3LBr9jluWUlR0/edit#gid=0

**Comment by tukkek** [comment]
This seems redundant to [the Debigare list of randomizers](https://www.debigare.com/randomizers/). The size of both projects is relatively similar and it would seem unwise to start adding randomizers here from  an effort, utility and presentation point of views.

It would be nice to have a FOSS version of Debigare's list but that would most likely be a better idea as its own project, not as an appendix to OSGC.
